### PR TITLE
Fabo/get network info from api

### LINKED ISSUE #3402: Address prefixes should be received from API
**Is your feature request related to a problem? Please describe.**

To be able to scale networks without needing to update the client we should deliver as much data from the API as possible. Currently the bech 32 prefixes for the Ledger handling are in config.js. These should be received from the API (networks query). Same goes for identifying valid addresses in the explore input.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by faboweb**
Closes #3402

Needs https://github.com/luniehq/lunie-api/pull/278 to be released first

Queries information about the used ledger app and address creator from the API so we don't need to have this information in the FE code (to scale better)

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Now it is working! :wink: And I love the whole refactor :+1:

**Comment by Bitcoinera** [comment]
I'm OK merging this. What about you @jbibla ?

File: src/components/common/TmSessionHardware.vue [hunk 1]
```diff
@@ -135,7 +135,7 @@ export default {
       this.status = `detect`
       this.address = null
       try {
-        this.address = await getAddressFromLedger(this.networkId)
+        this.address = await getAddressFromLedger(this.networkId, this.$apollo)
```
Comment (jbibla) [inline src/components/common/TmSessionHardware.vue]: why do we have to pass `$apollo` as an argument?

Comment (Bitcoinera) [reply src/components/common/TmSessionHardware.vue]: it needs it to query for the address prefix in `ledger.js`, lines 48-59. 
Before we were using the hardcoded dictionary and now we are using the API.

**Pull request merged**